### PR TITLE
Workflows: Update Playground blueprint with repo and branch/tag names.

### LINKED ISSUE #141: Change blueprint to based on the branch name dynamically
Change blueprint to based on the branch name dynamically. Probably a GitHub action would do this on a merge for all bransch and release tags

The current blueprint for the playground is hardcoded to retrieve the AU plugin from the playground-ready branch.  Change so it point  to itself.

https://github.com/aspirepress/AspireUpdate/blob/main/assets/playground/blueprint.json

This change would allow for easier testing of older releases in relation to specific issues across all branches on playground.

https://aspirepress.slack.com/archives/C07Q88M2KQF/p1730827139397469

### PR DESCRIPTION
**Pull request by costdev**
# Pull Request

## What changed?

- A new workflow has been added which will update the Playground blueprint with the current repository and branch/tag names.
    - Handling the repository name means that even renaming the repository will be covered for each branch on their next push.
    - A commit is only attempted if the workflow made a change.
    - Uses the same `tags` entry as the `releases.yml` workflow for consistency.

## Why did it change?

Playground testing is currently only possible on the `playground-ready` branch. By updating the playground blueprint for the `main` branch and tags, these can be tested too.

## Did you fix any specific issues?

Fixes #141 

## CERTIFICATION

By opening this pull request, I do agree to abide by the [CODE OF CONDUCT](https://github.com/aspirepress/.github/CODE_OF_CONDUCT.md) and be bound by the terms of the [Contribution Guidelines](https://github.com/aspirepress/.github/CONTRIBUTING.md) in effect on the date and time of my contribution as proven by the revision information in GitHub. I also agree that any previous contributions shall be deemed subject to the terms of the ersion in effect on the date and time of this pull request, or any future revisions for pull requests I may submit.

Further, I certify that this work is my own, is original, does not violate the intellectual property of any other person or entity, and I am not violating any license agreements or contracts I have with any person or entity. Finally, I agree that this code may be licensed under any license deemed appropriate by AspirePress, including but not limited to open source, closed source, proprietary or custom licenses, and that such license terms neither violate my rights or my copyright to this code.

### CODE REVIEW

File: .github/workflows/playground-blueprint.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    tags:
+      - "**"
+
+jobs:
+  update-blueprint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```
Comment (afragen) [inline .github/workflows/playground-blueprint.yml:17]: Latest version of actions/checkout@v4

Comment (costdev) [reply .github/workflows/playground-blueprint.yml]: Fixed in [502ce31](https://github.com/aspirepress/AspireUpdate/pull/162/commits/502ce31973e241e0a067b8a2c4fef1572ca08ecc).

**Review by asirota — APPROVED** [review]

**Comment by asirota** [comment]
Well done @costdev thank you very much!

**Review by asirota — APPROVED** [review]
Just to confirm this will be updated on every branch whenever main or playground-ready is pushed. Is that right?

**Comment by costdev** [comment]
Not all at once, no. This will update just on the single branch that receives a push, which is currently limited to main, playground-ready, or a tag.

**Comment by asirota** [comment]
I see. And any new branches we could adjust the file to include that branch in the workflow. Got it. Thanks again.

**Comment by asirota** [comment]
Looks like the automation failed to complete. Have a look @costdev please.

**Pull request merged**